static/panes/pp-view.ts

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) 2017, Simon Brand
```

Comment:
Can you update this copyright please? It's pretty important we don't attribute copyright inappropriately here :)